testsuite/classlibrary/TestOSCFunc.sc

```diff
@@ -0,0 +1,32 @@
+TestOSCFunc : UnitTest {
+
+	test_argSizeMatchingSucceeds {
```

Comment:
I'm nitpicking, but I think the tests would be more clearly named as "test_argSizeMatching_succeeds" and "test_argSizeMatching_fails".

Reply:
Really? I find that a slightly odd mix of camel case and underscore. Okay there was already test_ at the beginning but I understood that as a convention to support the slightly bodgy way UnitTest identifies test methods.

But I don't mind if you change them, and thanks for reviewing

Reply:
This is covered in the guide I wrote, is already used throughout the test suite, and I found it to be something that others recommended in their guides. Descriptive test names often have 2 or 3 components, each of which may be multiple words, so there's no other way to make that clear

Reply:
If you disagree with that please make a case for it on the ML or on the gist, it's still a WIP!

Reply:
Ah sorry!

Personally I don't find that much more readable than just camel case, but I'm okay with it. I don't generally like when original conventions are diluted with new ones, but this is just the test suite after all, so it could be a special case. I'd prefer not to add lots of thing_camelCase_etc in Common though.

Reply:
Yes definitely, this is a convention that should only be kept in test code, in Common almost all (hopefully all) method names are free from underscores